_data/other.yml

```diff
@@ -279,6 +279,16 @@ websites:
       img: promoter.png
       tfa: No
       twitter: Promoter_io
+      
+    - name: Repairshopr
+      url: http://www.repairshopr.com/
```

Comment:
Are you sure? It doesn't redirect for me.
![screenshot](https://puu.sh/rfYat/cf6c16ec9c.png)